src/energy_systems/control_modules/default.jl

```diff
@@ -0,0 +1,45 @@
+"""
```

Comment:
Why are the parameters for the load/unload control of memories not included in the parameter list? I haven't looked deeply into the new control mechanism, maybe I just overlooked them...

Reply:
The entry `control_modules` is a list of dictionaries, each containing the parameters for that particular module. However some parameters concern not a single module but the overall control behaviour, in particular parameters defining how the results of all control modules are aggregated. This is the reason for the additional dictionary under `control_parameters`.

As to why the storage un-/loading is in the default module... I think this is an artifact from how I restructured the code from the previous implementation. Perhaps it would make more sense to have these parameters (and the `consider_*` ones) as part of `control_parameters`, making the default module superfluous. I will think about it.